masterlist.yaml

```
@@ -6910,6 +6910,19 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/18571/' ]
     after: [ 'Requiem.esp' ]
 
+  - name: 'Containers and Leveled Lists Fixes.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/26575/' ]
+    group: *economyGroup
+    after:
+      - 'LoreWorthyBandits.esp'
+      - 'Weapon AF.esp'
```

Looks fine, it needs an after for Morrowloot Ultimate as well.

```suggestion
      - 'LoreWorthyBandits.esp'
      - 'MLU.esp'
      - 'Weapon AF.esp'
```